src/Components/dashboard/route/Skeletons/SkeletonElement.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import Skeleton from'react-loading-skeleton';
+import {
+    Box,
+    Grid,
+    ListItemButton,
+    ListItemText,
+    Typography,
+  } from "@mui/material";
+<<<<<<< HEAD
```

Comment:
You have to remove these un-resolved conflicts in this file

Reply:
Look like I missed this ones, I have resolved them